### PR TITLE
remove token on signout

### PR DESCRIPTION
**Pull request by yujinjcho**
no ticket for this

### CODE REVIEW

File: src/SignOut.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import Container from 'react-bootstrap/lib/Container';
+
+import auth from './auth';
+
+class SignOut extends Component {
+
+  componentDidMount() {
+    auth.removeFromHeader();
```
Comment (achiu8) [inline src/SignOut.js]: Actually, might be better to do this immediately after success of the sign-out request.

Comment (yujinjcho) [reply src/SignOut.js]: are you referring to `sign-out` on the play app?

Comment (achiu8) [reply src/SignOut.js]: No I mean wherever in the client app we actually make the request to the `/sign-out` endpoint--like probably from a nav link or something--in the `.then` callback on success.

Comment (yujinjcho) [reply src/SignOut.js]: oh got it. i can try to add navbar

**Comment by yujinjcho** [comment]
yo do you know what's going on with these errors? I'm copying pretty much directly from https://react-bootstrap.github.io/components/navbar/#navbars-mobile-friendly

screen shot 2019-01-19 at 2 45 07 pm

**Comment by yujinjcho** [comment]
how this

**Comment by achiu8** [comment]
Looks good.

**Pull request merged**